jgiven-maven-plugin/build.gradle

```diff
@@ -22,27 +22,31 @@ dependencies {
 }
 
 test {
-    useJUnitPlatform();
+    useJUnitPlatform()
 }
 
-task copyClasses(type: Copy, dependsOn: compileJava) {
+tasks.register('copyClasses', Copy) {
+    dependsOn(compileJava)
     from 'build/classes/java/main'
     into 'build/maven/target/classes'
 }
 
-task generatePom(type: Copy, dependsOn: copyClasses) {
+tasks.register('generatePom', Copy) {
+    dependsOn(copyClasses)
     from 'src/main/maven'
     into 'build/maven'
     filesMatching("**/pom.*") {
         expand(version: project.version)
     }
 }
 
-task generateMavenPlugin(type: CrossPlatformExec, dependsOn: generatePom) {
+tasks.register('generateMavenPlugin', CrossPlatformExec) {
+    dependsOn(generatePom)
     // currently it seems to be the more or less only clean solution
     // to generate a plugin.xml file to use maven directly
     // if anyone has a better solution please let us know!
-    buildCommand 'mvn', '-nsu', '-U', '-f', 'build/maven/pom.xml', 'plugin:descriptor'
+    buildCommand 'mvn', '-nsu', '-U', '-f', 'build/maven/pom.xml', 'plugin:descriptor', '--batch-mode',
+            '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
```

Comment:
Why not debug?

Reply:
because I want less output for the maven plugin specifically
I usually clogs our release logs with stupid stuff

Reply:
Alright! Let's merge!